### DISCUSSION TITLE
Is there any way to capture logs when level=0？

### DISCUSSION THREAD

**Discussion by chenqinggang001** [seleniumbase/SeleniumBase#1439] [Q&A] [answered]
Is there any way to capture logs when level=0？ # I used `-- html` and the log does not appear in the report

About Visual Tests whether there is a new iteration plan in the future？

**Comment by mdmintz** [comment, marked as answer]
The HTML reports capture stack traces for failing tests. (``--html=report.html``). For each failing test, there will also be an entry in the ``latest_logs/`` folder. That's the logging system.

The current Visual Testing solution has all the functionality that's needed: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/visual_testing/ReadMe.md
